vendor/src/github.com/docker/libcontainer/CONTRIBUTING.md

```diff
@@ -42,7 +42,7 @@ Like the previous command, you can run this on OSX too!
 
 ### Building libcontainer directly
 
-> Note: You should add the `vendor` directory to your GOPATH to use the vendored libraries
+> Note: You should add the `vendor` directory to your GOPATH to use the vendor libraries
```

Comment:
vendored is correct in this context - meaning adding a particular version of an external library to your project's revision control system.